stratum/lib/utils.cc

```diff
@@ -270,4 +271,14 @@ ::util::Status CreatePipeForSignalHandling(int* read_fd, int* write_fd) {
   return ::util::OkStatus();
 }
 
+::util::Status ValidateIETFYangHexString(std::string input) {
```

Comment:
Where are we using this function?
I understood from Derek that this is to transport the encryption key via Protobuf for IPSEC. If it's specific to how we transport the key for ES2ky, then we should introduce a functionsn in utils module under tdi instead of modifying common stratum lib utils